src/integrationTest/java/org/opensearch/node/PluginAwareNode.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Does this file come from floragunn repo or is it a new file ? If it is a new file we should remove the floragunn license section